src/Abp.AspNetCore/AspNetCore/Mvc/Antiforgery/AbpAutoValidateAntiforgeryTokenAuthorizationFilter.cs

```diff
@@ -32,7 +32,8 @@ protected override bool ShouldValidate(AuthorizationFilterContext context)
             }
 
             //Always perform antiforgery validation when request contains authentication cookie
-            if (context.HttpContext.Request.Cookies.ContainsKey(_cookieAuthenticationOptions.Cookie.Name))
```

Comment:
@acjh  `_antiforgeryOptions.Cookie.Name` Is it necessary to also perform a non-null check?